.rhdh/bundle/manifests/rhdh-operator.csv.yaml

```diff
@@ -357,4 +357,4 @@ spec:
     name: Red Hat Inc.
     url: https://www.redhat.com/
   version: 1.3.2
-  replaces: rhdh-operator.v1.2.4
+  replaces: rhdh-operator.v1.2.5
```

Comment:
Need to change this to the version of 1.2 that was released on the fast channel - 1.2.0 or something

Reply:
```suggestion
  replaces: rhdh-operator.v1.2.3
```